src/main/java/org/basepom/mojo/duplicatefinder/classpath/ClasspathDescriptor.java

```diff
@@ -73,6 +73,12 @@
         ".*overview\\.html$"
         ));
 
+    private static final MatchPatternPredicate DEFAULT_IGNORED_CLASS_PREDICATE = new MatchPatternPredicate(Arrays.asList(
+        // this regex matches nothing at all, just need something as a place holder
```

Comment:
huh, what? This should either be empty or, if MatchPatternPredicate for whatever reason does not support an empty array, it should be fixed.